extensions/adapters/vector/src/main/java/mil/nga/giat/geowave/adapter/vector/plugin/GeoWaveGTDataStoreFactory.java

```diff
@@ -134,9 +134,8 @@ public boolean canProcess(
 			return true;
 		}
 		catch (GeoWavePluginException e) {
-			LOGGER.error(
```

Comment:
good catch - if anything this should have been at the info level, but definitely not error